coding/unit_test_coverage.rst

```diff
@@ -13,25 +13,26 @@ executing instrumented unit tests which records the complete execution path
 through the code and then calculating metrics indicative of the coverage
 achieved during execution.
 
-Coverage analysis examines the output of a code instrumented to record every
+Coverage Analysis examines the output of a code instrumented to record every
 line executed, every conditional branch taken, and every block executed. Then
-using static information such as total number of lines of code, branches, and
-blocks; lists of functions; and class methods, it generates metrics on:
+using static information such as total number of: lines of code, branches, and
+blocks; lists of functions and class methods, it generates metrics on:
```

Comment:
Not my sentences! :) I just happened to notice that unit_test_coverage and unit_test_policy had identical paragraphs (from the Confluence days) and tried to consolidate them here. Could do with some editing, indeed.